pkg/cmd/target.go

```diff
@@ -566,7 +566,14 @@ func targetShoot(targetWriter TargetWriter, shoot gardencorev1beta1.Shoot, reade
 	gardenClient, err := target.K8SClientToKind(TargetKindGarden)
 	checkError(err)
 	seedKubeconfigSecret, err := gardenClient.CoreV1().Secrets(seed.Spec.SecretRef.Namespace).Get(seed.Spec.SecretRef.Name, metav1.GetOptions{})
-	checkError(err)
+
+	// temporary solution , will clear up code in ticket move get seed  out of targetShoot method
```

Comment:
```suggestion
	// temporary solution , will clean up code in ticket move get seed out of targetShoot method #269
```
added reference to the issue